### DISCUSSION TITLE
why my soc page  don't  have "administrator --> configuration" ?

### DISCUSSION THREAD

**Discussion by apt668** [Security-Onion-Solutions/securityonion#14054] [Unsupported Versions] [answered]
my version:2.3.300
install type:  standalone in my local environment
i see  some  buttons , such as   "configuration" 、“Users”、“Grid Members" in  Administration. 
the url  is   "https://docs.securityonion.net/en/2.4/strelka.html"
why my soc page just have "Users" button  ?
![Uploading 12.png…]()

**Comment by TOoSmOotH** [comment, marked as answer]
All 2.3 versions of the product have been end of life for quite a while now. The web interface for configuration was a key feature for 2.4.
